### PR TITLE
return value

### PR DESCRIPTION
**Pull request by ckritzinger**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -123,11 +124,12 @@ <h1>Oops!</h1>
                 <label class="amountcontainer" for="email">Reference (Optional):</label>
                 <input type="text" id="reference" name="reference" onchange="setOrderId()" placeholder="order ref">
             </div>
-            <form id="payform" action="https://pay.cryptoqr.net/" method="GET" class="mb-1">
+            <form id="payform" action="http://localhost:5173/" method="GET" class="mb-1">
```
Comment (carelvwyk) [inline index.html]: revert

File: index.html [hunk 2]
```diff
@@ -89,7 +89,8 @@
             const urlParams = new URLSearchParams(window.location.search);
             return urlParams.get(param);
         }
-        function setMerchantCode() {
+        function init() {
+            document.getElementById('return').value = window.location.href;
```
Comment (carelvwyk) [inline index.html]: This this need to be encoded?

File: index.html (hunk 1 above)
Comment (ckritzinger) [reply index.html]: thanks

File: index.html (hunk 2 above)
Comment (ckritzinger) [reply index.html]: ah, yeah it seems so.

Comment (ckritzinger) [reply index.html]: done

**Pull request merged**